tests/extendrtests/src/rust/src/lib.rs

```diff
@@ -22,40 +20,53 @@ fn do_nothing() {
 // Convert a double scalar to itself
 // x a number
 #[extendr]
-fn double_scalar(x: f64) -> f64 { x }
+fn double_scalar(x: f64) -> f64 {
+    x
+}
 
 // Convert an int scalar to itself
 // x a number
 #[extendr]
-fn int_scalar(x: i32) -> i32 { x }
+fn int_scalar(x: i32) -> i32 {
+    x
+}
```

Comment:
These formatting changes were unintentional, I hooked `cargo fmt` to OnSave of my CLion, so once I edited this file, it automatically applied formatting and I did not notice it.